packages/teams-js/src/public/dialog.ts

```diff
@@ -101,7 +101,89 @@ export namespace dialog {
     }
   }
 
+  function updateResizeHelper(apiVersion = 'v1', dimensions: DialogSize): void {
```

Comment:
Can you explain what the benefit of moving these implementations out into helper funtions is? It doesn't seem to be adding much value that we wouldn't get by just adding apiVersion into the original implementations, and it makes the code harder to follow because the function signatures and implementations are in separate places.

Reply:
The code is moved out as a helper to resolve the difficulty of sending out different api version number.

In previous design, sendMessageToParent(WithVersion) API is embedded into updateResize function. And this function is used inside of Task capability.

Reply:
@TrevorJoelHarris, those helper functions are not in the proper files. I have moved them from public/dialog file to internal/dialogUtil file. I think it will make more sense to people now because helpers are used in both Task capability and Dialog capability.